sqlite/sqlite/src/androidMain/kotlin/androidx/sqlite/db/SupportSQLiteCompat.android.kt

```diff
@@ -91,6 +91,9 @@ public class SupportSQLiteCompat private constructor() {
     }
 
     /** Helper for accessing functions that require SDK version 21 and higher. */
+    @Deprecated(
```

Comment:
This can be removed as it is library_group scope and this library is an atomic group 

https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:libraryversions.toml;l=307

Same with the one below

Reply:
Done 👍🏻
What does it imply that it is an atomic group?

Reply:
It means that all libraries in this maven group are expected to be used at the same version and we enforce that using Gradle constraints.

You can read a bit more about it in `Same-version (atomic) groups` section in 
https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:docs/api_guidelines/modules.md